docs/deploy/self_hosted_github_runners.md

```diff
@@ -0,0 +1,106 @@
+# Self Hosted GitHub Runners
+
+A self-hosted runner is a system that you deploy and manage to execute jobs from GitHub Actions on GitHub.com.
+This can be useful for
+
+* running automation scripts
+* deploying production / pre-production servers
+
+You will need to be an adminstrator to configure a self-hosted runner
+
+## Create Self-Hosted Runner
+
+You can add an actions runner from the settings page (change the organization/project information)
```

Comment:
Not _quite_ true. The Settings page provides the instructions and token needed to create an actions runner. The creation/configuration happens on the server.

Reply:
Clarified that they give you documentation to set it up, not actually setting it up.  Please review new language.

<img width="891" alt="Screen Shot 2022-06-28 at 19 08 03" src="https://user-images.githubusercontent.com/48086/176318899-4bafab65-9b13-47d4-b065-e948dff3363c.png">